apps/front/src/components/atoms/Alert.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { InputHTMLAttributes } from 'react';
+
+interface AlertProps {
+    alertType?: 'alert' | 'alert-success' | 'alert-error' | 'alert-warning' | 'alert-info';
+    label?: string;
+    // daisyUIのbtnプロパティ
+    btnType?: 'btn-success' | 'btn-error' | 'btn-warning' | 'btn-info';
+    btnLabel?: string;
+    btnDisplay?: boolean;
```

Comment:
```suggestion
    btnDisplay?: boolean;
    onClick?: () => void;
```
アラート内のボタン押した時のonClickを外から呼び出せるようにしておいてほしい！！

Reply:
追加しました！